packages/api-core/src/domain-email/fetchers.ts

```diff
@@ -11,9 +11,9 @@ type EmailForwarder = {
 
 type EmailForwardersResult = {
 	type: 'forward';
-	forwards: EmailForwarder[];
+	forwards: null | EmailForwarder[];
```

Comment:
Nit: The "Happy path first" is typically used as a TypeScript standard pattern. I would expect `null` to come after `EmailForwarder[]`.